packages/application/src/index.ts

```diff
@@ -613,7 +714,20 @@ namespace Private {
     /**
      * The function which activates the plugin.
      */
-    readonly activate: (app: any, ...args: any[]) => any;
+    readonly activate: (
+      app: Application<Widget>,
+      ...args: Token<any>[]
```

Comment:
We don't pass `Token` instances into the `activate` function, we pass the resolved item (which can by `any` thing).

Reply:
True, thanks I was trying to improve typing but yep, that change is definitely inappropriate.